### PR TITLE
fix logistic regression ci failure when spark 3.3.0 is used

### PR DESCRIPTION
**Pull request by lijinf2**

### CODE REVIEW

**Comment by lijinf2** [comment]
build

**Comment by lijinf2** [comment]
build

**Comment by lijinf2** [comment]
build

File: python/tests/test_logistic_regression.py [hunk 1]
```diff
@@ -1132,13 +1132,16 @@ def test_compat_one_label(
...
+            spark_v33_msg = (
+                f"Classification labels should be in [0 to -1]. Found 4 invalid labels."
+            )
 
             try:
                 blor_model = blor.fit(bdf)
                 assert False, "There should be a java exception"
             except Py4JJavaError as e:
-                assert msg in e.java_exception.getMessage()
+                assert spark_v34_msg or spark_v33_msg in e.java_exception.getMessage()
```
[the remark this answers is not in the document]
Comment (lijinf2) [reply python/tests/test_logistic_regression.py]: Good catch. Just pushed the revised code.

**Review by eordentlich — APPROVED** [review]
👍

**Pull request merged**